### ISSUE TITLE
why src/client is empty

### ISSUE THREAD

**Issue by omgbbqhaxx** [jsoneaday/solana-example-helloworld#7]
why src/client is empty can you help me

**Comment by DhiaMnasser** [comment]
that's what happens when you git init inside a folder of a project, I'm waiting for the source code too,
but you can download the code from the [original project](https://github.com/solana-labs/example-helloworld) and follow his instruction in the video.

**Comment by jsoneaday** [comment]
Yeah I screwed that up but it's not as bad as it seems. The video source code is based on the Solana hello world example, meaning most of the code is from that project. Found here https://github.com/solana-labs/example-helloworld.

Just clone that and you'll be able to do a lot less typing. The subsequent videos will have more custom code.
